<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLinkInit</class>
 <widget class="QWidget" name="QLinkInit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Options</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QSlider" name="_height_slider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_width_slider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="_kind_slider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="_width_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_height_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_kind_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="width">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**宽度**</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="height">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**高度**</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kind">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>**方块种类数量**</string>
   </property>
   <property name="textFormat">
    <enum>Qt::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_cancel_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_ok_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认并开启新游戏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>QLinkInit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_ok_button</sender>
   <signal>clicked()</signal>
   <receiver>QLinkInit</receiver>
   <slot>startNewGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_height_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QLinkInit</receiver>
   <slot>changeHeightValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_width_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QLinkInit</receiver>
   <slot>changeWidthValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_kind_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QLinkInit</receiver>
   <slot>changeIconKind(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startNewGame()</slot>
  <slot>changeWidthValue(int)</slot>
  <slot>changeHeightValue(int)</slot>
  <slot>changeIconKind(int)</slot>
 </slots>
</ui>
